tests/micropython/ringio.py

```diff
@@ -45,4 +45,4 @@
     # Size must be int.
     micropython.RingIO(None)
 except TypeError as ex:
-    print(ex)
+    print(repr(ex)[:24], repr(ex)[-9:])
```

Comment:
This seems a bit opaque... what about `print("to int" in repr(ex)) # error string depends on port` or even `pass`?

Reply:
It definitely needs to print something here, otherwise the test will pass even if the exception is not raised.

I think some other tests do this string mangling, but here it's not really necessary, so I changed it to `print(type(ex))`, which is the same on all builds.

Retested on PYBD-SF2 and ESP8266_GENERIC.

Reply:
Looks good to me!